tests/Aspire.Hosting.MySql.Tests/MySqlFunctionalTests.cs

```diff
@@ -27,7 +28,7 @@ public class MySqlFunctionalTests(ITestOutputHelper testOutputHelper)
     [RequiresDocker]
     public async Task VerifyWaitForOnMySqlBlocksDependentResources()
     {
-        var cts = new CancellationTokenSource(TimeSpan.FromMinutes(3));
+        var ct = new CancellationTokenSource(TestConstants.ExtraLongTimeoutTimeSpan).Token;
```

Comment:
I know it wasn't before - but shouldn't we be disposing the `CancellationTokenSource`?

Reply:
oh! yes

Reply:
why isn't a roslyn analyzer complaining...